tests/unit/CertificateGenerationService.unitTest.ts

```diff
@@ -960,28 +989,293 @@ describe("Certificate Generation Service", () => {
         jest.clearAllMocks();
       });
     });
+    describe("Welsh feature flags", () => {
```

Comment:
Inside of this describe would be good to have a before each going on here, same for after each as there seems to be common functionality in all of the tests. E.g. 

      let certGenSvc: CertificateGenerationService;

      beforeEach(() => {
        certGenSvc = new CertificateGenerationService(null as any, new LambdaService(new Lambda()));
      });

      afterEach(() => {
        jest.resetAllMocks();
      });